requirements/requirements.txt

```diff
@@ -8,3 +8,4 @@ torch
 torchvision
 wget
 wrapt
+PyYAML
```

Comment:
If PyYAML is now in requirements.txt, please remove it from all other requirements files.

Reply:
done